### PR TITLE
[portsyncd]: Skip empty lines

### PR DESCRIPTION
**Pull request by stcheng**
If the length of the line is 0, skip parsing this line.

Signed-off-by: Shu0T1an ChenG <shuche@microsoft.com>

### CODE REVIEW

File: portsyncd/portsyncd.cpp [hunk 1]
```diff
@@ -205,6 +205,11 @@ void handlePortConfigFile(ProducerTable &p, string file)
     string line;
     while (getline(infile, line))
     {
+        if (line.size() == 0)
```
Comment (pavel-shirshov) [inline portsyncd/portsyncd.cpp]: what if line has a space in it?

Comment (jleveque) [reply portsyncd/portsyncd.cpp]: Every line in port_config.ini should contain spaces, as they are space-delimited. What is your concern? This code should only skip empty lines. Lines containing spaces should have a length > 0.

Are you asking about lines which *only* contain whitespace?

Comment (pavel-shirshov) [reply portsyncd/portsyncd.cpp]: If a line has a length equal to 1 will that line work for us?
Should we better skip a line which contains from 0 to many spaces?

Comment (qiluo-msft) [reply portsyncd/portsyncd.cpp]: This is a feature/fix requested by me. Normally I prefer a config file end with a new line character.

**Comment by stcheng** [comment]
closed

**Pull request closed without merging**